client-src/elements/chromedash-app.js

```diff
@@ -240,6 +284,33 @@ class ChromedashApp extends LitElement {
     return url;
   }
 
+  renderContentAndSidebar() {
+    const wide = (this.pageComponent &&
+                  this.pageComponent.tagName == 'CHROMEDASH-ROADMAP-PAGE');
+    if (wide) {
+      return html`
+        <div id="content-component-wrapper" wide>
```

Comment:
What is the `wide` here?

Reply:
It is an attribute used to trigger a CSS rule to make the main content of the page wide on certain pages, like the roadmap page.  Those pages won't have a sidebar.   I suppose that it could be done with a CSS class instead of a separate attribute, but this is existing code so I kept it as-is in this PR.